.github/workflows/main.yml

```diff
@@ -159,6 +159,20 @@ jobs:
     - name: Style check
       run: nix-shell --pure --run "make check"
 
+
+  python-test-style:
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - uses: psf/black@stable
```

Comment:
Surely this should be different now?

Reply:
It should be using `nix-shell` now, currently I have included a pyproject.toml in the root folder, is this okay? It is used to have some default settings when using black in the folder.